README.md

```diff
@@ -325,6 +326,26 @@ Navigate to [New Release Form](https://github.com/CDCgov/prime-simplereport/rele
 4. Click publish release
 5. Verify the changes are live by ensuring the deployed commit hash matches the commit hash on the release. This is done my going to `/app/static/commit.txt` and `/api/actuator/info`
 
+### Revert to a Previous Release
+
+1. Find the version tag for the release you want to revert to.
+2. Checkout that version and create a new branch
+    ```bash
+    $ git checkout
```

Comment:
Should this be `git checkout ${version_tag}`?

Reply:
Yest it should!
```suggestion
    $ git checkout ${version_tag}
```